docs/src/adoc/index.adoc

```diff
@@ -10,6 +10,7 @@
 :source-highlighter: coderay
 :source-language: asciidoc
 :imagesdir: images
+:docinfo: private
```

Comment:
I'm not sure I understand this change.  Is this not the default? 

https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/_includes/docinfo.adoc

> Setting docinfo with no value is equivalent to setting the value to private.

Reply:
done

Reply:
Can we not omit the key entirely?  (That was the change I expected to see)  Or is its presence important?

Reply:
I tried it, doesn't work. The doc implies as much.